lib/site-utils.js

```diff
@@ -86,6 +88,13 @@ exports.processPrice = function processPrice(priceStringInput) {
     _.includes(priceString, GBP_SYMBOL_PREFIX)) {
     logger.log('found gbp in price, converting to number...');
     price = accounting.unformat(priceString);
+  } else if (_.includes(priceString, INR_SYMBOL_PREFIX) ||
+    _.includes(priceString, INR_SYMBOL_PREFIX)) {
```

Comment:
Do my eyes deceive me, or is that the same check with an or statement between them?

```
(_.includes(priceString, INR_SYMBOL_PREFIX) || _.includes(priceString, INR_SYMBOL_PREFIX))
```
Maybe a copy/paste thing, but do you mind removing the second check if that is not needed?